addons/event/models/event_event.py

```diff
@@ -241,7 +241,6 @@ def _default_question_ids(self):
             ('A6', 'A6'),
             ('four_per_sheet', '4 per sheet'),
             ('96x82', '96x82mm (Badge Printer)'),
-            ('96x134', '96x134mm (Badge Printer)')
```

Comment:
What happens when we remove a selection option in stable (can we?)
Does it need an upgrade script?

Reply:
you cannot change model in stable

Reply:
okay, will retarget to master then